android/src/main/java/bbct/android/common/activity/About.java

```diff
@@ -53,5 +53,4 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
 
         return view;
     }
-
```

Comment:
I like this. Let's consistently remove any blank lines between two closing braces. Don't go out of your way to do it, but feel free to do so whenever you edit a file for some other change.

Reply:
okay